dev/workflow/rocoto/gfs_tasks.py

```diff
@@ -2921,7 +2926,12 @@ def earc_tars(self):
             if self.options['do_jediocnvar']:
```

Comment:
One line above this, I missed adding a `deps.append` call.  I think this should be

```suggestion
            deps.append(rocoto.add_dependency(dep_dict))
            if self.options['do_jediocnvar']:
```

Reply:
@DavidHuber-NOAA : This would make the `enkfgfs_earc_tars` jobs dependent on both the `esfc` and `ecmn` jobs, correct?  If so, yes this is what we need.  I think it only looks like it works as it is now because `esfc` and `ecmn` run side-by-side and the `ecmn` jobs tend to run longer than `esfc`.

Reply:
Thinking more on this... Sorry I didn't look more closely at https://github.com/NOAA-EMC/global-workflow/pull/3660, but the added `ecmn` dependency should be applicable for ATM-only cases as well, i.e. I don't think it needs to be behind the `do_jediocnvar` IF block.  This is separate from the dependencies on `ocnanalecen` and `marineanlfinal`.